README

```diff
@@ -20,14 +20,15 @@ sudo packer -S mozart-git
 Building mozart for Ubuntu/Debian (x64).
 ----------------------------------
 $ sudo apt-get install emacs flex bison tk-dev build-essential g++-multilib zlib1g-dev:i386 libgmp-dev:i386
-$ mkdir -p ~/dev/mozart
-$ cd ~/dev/mozart
```

Comment:
It would be useful to keep these so it's clear we don't expect e.g. a `~/build` directory.

Reply:
Thanks, fixed in a new pull request.